examples/vision/mixUp.py

```diff
@@ -0,0 +1,219 @@
+"""
+Title: mixup augmentation for image classification
+Author: [Sayak Paul](https://twitter.com/RisingSayak)
+Date created: 2020/03/06
+Last modified: 2020/03/06
+Description: How to implement mixup to augment images for image classification problems.
+"""
+"""
+## Introduction
+"""
+
+"""
+mixup is a *domain-agnostic* data augmentation technique proposed in [mixup: Beyond
+Empirical Risk Minimization](https://arxiv.org/abs/1710.09412) by Zhang et al. It's
+implemented as the following -
+
+* $\tilde{x}=\lambda x_{i}+(1-\lambda) x_{j}$, where $x_{i}$ and $x_{j}$ are input
+features
+* $\bar{y}=\lambda y_{i}+(1-\lambda) y_{j}$, where $y_{i}$ and $y_{j}$ are one-hot
+encoded labels
+
```

Comment:
Suggestion: expand this to help the users understand what each of the variables mean, even though we go into more detail in the section where you define the function

```suggestion
* $\tilde{x}=\lambda x_{i}+(1-\lambda) x_{j}$, where $x_{i}$ and $x_{j}$ are input
features
* $\bar{y}=\lambda y_{i}+(1-\lambda) y_{j}$, where $y_{i}$ and $y_{j}$ are one-hot
encoded labels

(Note that $lambda x_{i}$ and $lambda y_{i}$ are values ranging from 0 to 1 and are sampled
from the [Beta distribution](https://en.wikipedia.org/wiki/Beta_distribution).)
```